types/interceptors.d.ts

```diff
@@ -20,7 +20,7 @@ declare namespace Interceptors {
     maxTTL?: number
     maxItems?: number
     lookup?: (hostname: string, options: LookupOptions, callback: (err: NodeJS.ErrnoException | null, addresses: DNSInterceptorRecord[]) => void) => void
-    pick?: (origin: URL, records: DNSInterceptorOriginRecords, affinity: 4 | 6) => DNSInterceptorRecord
+    pick?: (origin: URL, records: { records: DNSInterceptorOriginRecords }, affinity: 4 | 6) => DNSInterceptorRecord
```

Comment:
This looks like a bug in the implementation rather than the types..

Reply:
oh no…，what should we do? Change implementation will cause a breaking change but fix types is odd.